### PR TITLE
Add mergify

### PR DESCRIPTION
**Pull request by ArmaanT**
Automating myself out of a job

### CODE REVIEW

**Comment by KeenWill** [comment]
> Automating myself out of a job

Time to become a luddite.

File: .github/mergify.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+pull_request_rules:
+  - name: automatic merge for CI config
+    conditions:
+      - author=dependabot[bot]
+      - files=.github/cdk/package.json
+      - check-success=Validate cdkactions manifests
+      - base=master
+    actions:
+      merge:
+        method: squash
```
Comment (KeenWill) [inline .github/mergify.yml:10]: Is squash really needed if dependabot makes one commit?

**Comment by brandonwang1** [comment]
> Automating myself out of a job

Can we make an Armaan CDK?

**Comment by eyingxuan** [comment]
> > Automating myself out of a job
> 
> Time to become a luddite.

There's always something to rewrite, no worries here.

**Comment by ArmaanT** [comment]
> Can we make an Armaan CDK?

No comment 

> There's always something to rewrite, no worries here.

In this case, I think it's just writing documentation for all of the things

File: .github/mergify.yml (hunk 1 above)
Comment (KeenWill) [reply .github/mergify.yml:10]: Ah makes sense

**Review by KeenWill — APPROVED** [review]
Let's deprecate Armaan

**Comment by ArmaanT** [comment]
Thanks, @KeenWill, it means a lot

**Pull request merged**